tests/integration/examples/test_host.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from adcm_aio_client.client import ADCMClient
+from adcm_aio_client.errors import ObjectDoesNotExistError
+from adcm_aio_client.objects import Cluster, Host, HostProvider
+
+pytestmark = [pytest.mark.asyncio]
+
+
+async def test_host(admin_client: ADCMClient, example_cluster: Cluster, example_hostprovider: HostProvider) -> None:
```

Comment:
Please add a brief description of the actions demonstrated in this test, such as in the test_bundle test.

Reply:
added